source/core/NumericString.cpp

```diff
@@ -1868,7 +1829,31 @@ VIREO_FUNCTION_SIGNATUREV(StringFormat, StringFormatStruct)
         format.AliasAssign(tempformat.Begin(), tempformat.End());
     }
     StringRef buffer = _Param(StringOut);
-    Format(&format, count, arguments, buffer);
+    Format(&format, count, arguments, buffer, null);
+    return _NextInstruction();
+}
+struct StringFormatWithErrStruct : public VarArgInstruction
+{
+    _ParamDef(StringRef, StringOut);
+    _ParamDef(StringRef, StringFormat);
+    _ParamDef(ErrorCluster, ErrClust);
+    _ParamImmediateDef(StaticTypeAndData, argument1[1]);
+    NEXT_INSTRUCTION_METHODV()
+};
+
+VIREO_FUNCTION_SIGNATUREV(StringFormatWithErr, StringFormatWithErrStruct) {
```

Comment:
The code appears to mostly a duplicate of StringFormat. Please create a function that both StringFormat and StringFormatWithErr call so they share code.